dashboard/app/controllers/api/v1/peer_review_submissions_controller.rb

```diff
@@ -1,37 +1,31 @@
 class Api::V1::PeerReviewSubmissionsController < ApplicationController
   authorize_resource class: :peer_review_submissions
-  # GET /api/v1/peer_review_submissions/index_escalated
-  def index_escalated
-    # Get a list of submissions that have been escalated
-    # This will return an array of objects containing
-    # - Submitter
-    # - Course name
-    # - Unit
-    # - Level name
-    # - Submission date
-    # - Date it was escalated
-    # - ID of review
+  # All calls here will return an array of objects containing
+  # - Submitter
+  # - Course name
+  # - Unit
+  # - Level name
+  # - Submission date
+  # - Date it was escalated
+  # - ID of review
 
-    escalated_submissions = Hash.new
+  def index
+    # Get a list of submissions that are currently open
+    submissions = Hash.new
 
-    escalated_reviews = PeerReview.escalated.limit(50)
-
-    escalated_reviews.each do |review|
-      plc_course_unit = review.script.plc_course_unit
+    case params[:filter]
+      when 'escalated'
+        reviews = PeerReview.escalated.limit(50)
+      when 'open'
+        reviews = PeerReview.where(reviewer: nil).limit(50)
+      else
+        reviews = PeerReview.all.limit(50)
```

Comment:
I imagine we're going to want to make this limit controllable by a query parameter at some point; any reason not to go ahead and do that now?